src/components/PlayPanel.tsx

```diff
@@ -77,6 +77,7 @@ const PlayPanel = () => {
     if (showCardsPopup) {
       timer = setTimeout(() => {
         setShowCardsPopup(false);
+        setTilesConqueredThisTurn([]);
```

Comment:
Why in the timeout ?

Reply:
To do it at the same time we close the pop up